public/js/buttonFunctions.js

```diff
@@ -61,46 +61,50 @@ let resetTableData = function() {
   }
   classesTable.setData(tableData.currentTermData.classes);
 
-  tableData.currentTermData.calcGPA = computeGPA();
-
+  tableData.currentTermData.calcGPA = computeGPA(tableData.currentTermData.classes);
   let GPA = tableData.terms[currentTerm].GPA;
   let calcGPA = tableData.terms[currentTerm].calcGPA;
 
   if (anyEdited()) {
     //fix the editing system in the if statement above to be true if any of the classes are edited
     if (currentTerm == "current") {
       $(".select-selected").css('padding', "5px 16px 5px 16px");
-      $(".select-selected").html("Current Quarter GPA: " + GPA + "<br>Calculated GPA: " + calcGPA);
+      $(".select-selected").html("Current Quarter GPA: " + GPA.percent + "<br>Calculated GPA: " + calcGPA.percent);
       $("#" + currentTerm).css('padding', "5px 16px 5px 16px");
-      $("#" + currentTerm).html("Current Quarter GPA: "  + GPA + "<br>Calculated GPA: " + calcGPA);
-      document.getElementById('gpa_select').options[0].innerHTML = "Current Quarter GPA: "  + GPA + "<br>Calculated GPA: " + calcGPA;
-      document.getElementById('gpa_select').options[1].innerHTML = "Current Quarter GPA: "  + GPA + "<br>Calculated GPA: " + calcGPA;
+      $("#" + currentTerm).html("Current Quarter GPA: "  + GPA.percent + "<br>Calculated GPA: " + calcGPA.percent);
+      document.getElementById('gpa_select').options[0].innerHTML = "Current Quarter GPA: "  + GPA.percent + "<br>Calculated GPA: " + calcGPA.percent;
+      document.getElementById('gpa_select').options[1].innerHTML = "Current Quarter GPA: "  + GPA.percent + "<br>Calculated GPA: " + calcGPA.percent;
 
     } else {
+	    tableData.currentTermData.calcGPA = computeGPA(tableData.currentTermData.classes,termConverter.indexOf(currentTerm));
```

Comment:
Why does computeGPA take two arguments here? It's signature seems to account for a single argument.

line 177 calculate_grade.js
```
function computeGPA(classes) {
 let sum = 0; // Sum of classes' grades
 let count = 0; // Number of classes
 let fourSum = 0; // Sum of classes' grades on 4.0 scale
 let fiveSum = 0; // Sum of classes' grades on 5.0 scale
```

Reply:
I don't know; I just removed the second argument in 
7aa8356